recipes/resolvo-cpp/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "resolvo-cpp" %}
+{% set version = "0.1.0" %}
```

Comment:
Should this recipe [export runtime requirements](https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#export-runtime-requirements) to encode version range for ABI stability?

Reply:
Yes, good catch! I added it.